spec/inertia/rspec_helper_spec.rb

```diff
@@ -114,4 +114,20 @@ def puts(thing)
       end
     end
   end
+
+  describe '.have_exact_props' do
+    context 'when shared props are wrapped in a callable', inertia: true do
+      it 'compares props with either string or symbol keys' do
+        get lamda_shared_props_path
+
+        expect_inertia.to have_exact_props({
+          someProperty: {
+            property_a: "some value",
+            'property_b' => "this value",
```

Comment:
This is taken directly from the bug report, with one tweak to ensure complete ambivalence towards the key type.